cli.js

```diff
@@ -248,7 +251,7 @@ const _isBaked = bundle => {
     return false;
   }
 };
-const _uploadPackage = async dataArrayBuffer => {
+const _uploadPackage = async (dataArrayBuffer, xrpkName) => {
```

Comment:
Was this a bug?

Reply:
I think so -- the linter showed `argv` inside the function was undefined. 

These two commits are bug-fixes as a result of linting:
- https://github.com/webaverse/xrpackage-cli/pull/19/commits/6f27456919701d2f849a178297bf8c8e032997c5
- https://github.com/webaverse/xrpackage-cli/pull/19/commits/d2c30dc085706d10ac5013b332373bf884d71c6e

They might be better in another PR? I just added them here because the linter pointed them out and thought they weren't too big of a change :/

Reply:
Nope, this is a great place to fix it 👍